tests/evm_tools/test_t8n.py

```diff
@@ -107,6 +106,18 @@ def t8n_tool_test(test_case: Dict) -> None:
             json_result["withdrawalsRoot"]
         ) == t8n_tool.hex_to_root(data["result"]["withdrawalsRoot"])
 
+    if "receipts" in data["result"]:
+        assert len(json_result["receipts"]) == len(data["result"]["receipts"])
+        for t8n_receipt, expected_receipt in zip(
+            json_result["receipts"], data["result"]["receipts"]
+        ):
+            assert t8n_receipt["gasUsed"] == expected_receipt["gasUsed"]
+            assert (
+                t8n_receipt["transactionHash"]
+                == expected_receipt["transactionHash"]
+            )
+            assert t8n_receipt["blockHash"] == expected_receipt["blockHash"]
```

Comment:
```suggestion
            assert t8n_receipt["blockHash"] == expected_receipt["blockHash"]
```

Since you have added `blockHash` to the receipt, you do not need to check this.